### ISSUE TITLE
consensus too slow

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion-em-xmipp#326]
It always seems to work 1-by-1 whether streaming or not which makes it very slow for non-streaming cases. Would be also nice to have batch support then.

**Comment by dmaluenda** [comment]
Yes @azazellochg, it was optimized for streaming and the performance for non streaming has been affected. It's in the TO-DO list.

Thanks!

**Comment by DStrelak** [comment]
Is it related to https://github.com/I2PC/xmipp/issues/150 ?
